jib-core/src/test/resources/json/loadmanifest.json

```diff
@@ -1 +1 @@
-[{"config":"config.json","repoTags":["testregistry/testrepo:testtag"],"layers":["layer1.tar.gz","layer2.tar.gz","layer3.tar.gz"]}]
```

Comment:
This should also keep the `[]` since without it it isn't a valid loadable manifest.